### DISCUSSION TITLE
All my button font size became smaller

### DISCUSSION THREAD

**Discussion by Legrand5382** [Clooos/Bubble-Card#745] [Questions about config, custom styles and templates]
Since beta 3 I lost all my custom font sized, is there a way to desactivate the new font auto ajustement? 

Is there a new way to change the font size on button ?

**Comment by Clooos** [comment]
Hi, indeed since v2.2.0 I've reduced the font size by 1px, but I will add the possibility to change that directly in the theme file with a unique variable.

In the meantime you can change it with this custom style in each cards:

```yaml
* {
  font-size: 14px !important;
}
```

**Reply by Matssa56** [reply]
> Hi, indeed since v2.2.0 I've reduced the font size by 1px, but I will add the possibility to change that directly in the theme file with a unique variable.
> 
> In the meantime you can change it with this custom style in each cards:
> 
> ```yaml
> * {
>   font-size: 14px !important;
> }
> ```

Hi, did you manage to put thé variable in the theme? Checked on latest release (not beta) and can't seem to find it through F12 in Chrome, can only see the font size as shown by your code. Indeed I wish to change the font size of the button title but also the button state / attribute / last changed.

Thanks again for the global variables, I'm using them to change the default color of the button and icon if it's on.
